xrpl_api/src/api/account_objects.rs

```diff
@@ -45,21 +46,6 @@ impl WithRequestPagination for AccountObjectsRequest {
     }
 }
 
-#[derive(Clone, Debug, Copy, Eq, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "snake_case")]
-pub enum AccountObjectType {
```

Comment:
Moved to type `ObjectType` that is shared with the `ledger` and `ledger_data` API methods